server.py

```diff
@@ -32,6 +32,22 @@ def get_shopcarts(id):
 
     return jsonify(message), rc
 
+######################################################################
+# DELETE A SHOPCART
+######################################################################
+@app.route('/shopcarts/<int:id>', methods=['DELETE'])
+def delete_shopcarts(id):
+    """
+    Delete a Shopcart
+    This endpoint will delete a Shopcart based on the id specified in the path
+    """
+    cart = Shopcart.find(id)
+    
+    if cart:
+        cart.delete()
+
```

Comment:
You need to send a message to the user if the cart he was deleting is not there. It's fine that it does not creates an error but maybe that he entered a wrong id and a simple message could really help a user in this case.

Reply:
According to the slides: 
```
If you DELETE a resource, it’s gone. If you DELETE it again, it’s still gone (i.e., do
not throw a 404 Not Found error!)
```

Are we following the professor's TDD repo or no? Let's not go overboard. `MVP`

Reply:
I did not say send an error. Just pass a message in the response instead of being empty.

We should follow the repo but as a programming guide not as a let's just copy everything and we are done.

Reply:
A delete request is supposed to return a 204 response with no content. I don't think sending back a message gives any benefit.

Now you have different 'success' paths for deleting that the user of the API needs to worry about. Also, in a real world scenario, you're giving malicious users too much information.

Reply:
Actually the malicious scenario is a good argument. I was being kind with the user hahaha we should be careful instead!